go/chat/boxer.go

```diff
@@ -858,20 +866,21 @@ func (b *Boxer) UnboxMessages(ctx context.Context, boxed []chat1.MessageBoxed, c
 	return unboxed, nil
 }
 
-// Can return (nil, nil) if there is no saved merkle root.
-func (b *Boxer) latestMerkleRoot() (*chat1.MerkleRoot, error) {
+// If no error then MerkleRoot is non-nil.
+func (b *Boxer) latestMerkleRoot(ctx context.Context) (*chat1.MerkleRoot, error) {
 	merkleClient := b.G().GetMerkleClient()
 	if merkleClient == nil {
 		return nil, fmt.Errorf("no MerkleClient available")
 	}
-	merkleRoot, err := merkleClient.LastRootInfo()
+	mr, err := merkleClient.FetchRootFromServer(ctx, time.Duration(0))
```

Comment:
Just to make sure I'm following, this wouldn't disrupt offline sending right? This will get done later at push time when the device comes back online?